### PR TITLE
[WIP/ENH] mne interface: SetupSourceSpace

### PR DESCRIPTION
**Pull request by vferat**
## Summary
Part of discussion in #3032

## List of changes proposed in this PR (pull-request)
Add **SetupSourceSpace** for **mne_setup_source_space command**.  [API](https://mne.tools/dev/generated/commands.html?highlight=command#mne-setup-source-space)

## Acknowledgment

- I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by effigies** [comment]
Hi @vferat, sorry to do this to you, but we just merged some pretty huge changes. Could you re-apply these changes on the latest `master`? You'll want to re-run `make specs` entirely, and we're now using the [Black](https://black.readthedocs.io/en/stable/installation_and_usage.html) formatter. You can run it manually, or use [pre-commit](https://pre-commit.com/) to have it automatically run every time you commit.

Please let me know if you need any help or clarifications.

File: nipype/interfaces/mne/base.py [hunk 1]
```diff
@@ -145,3 +146,122 @@ def _list_outputs(self):
                         mesh_paths.append(out_files)
         outputs["mesh_files"] = mesh_paths
         return outputs
+
+
+class SetupSourceSpaceInputSpec(FSTraitedSpec):
+    subject = Str(
```
Comment (satra) [inline nipype/interfaces/mne/base.py]:
```suggestion
    subject = traits.Str(
```

File: nipype/interfaces/mne/base.py [hunk 2]
```diff
@@ -145,3 +146,122 @@ def _list_outputs(self):
...
+        argstr='--subject %s',
+        mandatory=True,
+        desc='Subject name')
+    fname = File(
+        argstr='--src %s',
+        mandatory=False,
+        default=None,
+        desc='Output file name. Use a name <dir>/<name>-src.fif')
+    subject_to = Str(
```
Comment (satra) [inline nipype/interfaces/mne/base.py]:
```suggestion
    subject_to = traits.Str(
```

**Comment by satra** [comment]
@vferat - i fixed some of the merge conflicts online, but there are a couple that i missed. i posted as suggestions. if you can accept them, i think this will pass the tests.

**Pull request closed without merging**